### ISSUE TITLE
ArticleJson does not accept additional props

### ISSUE THREAD

**Issue by aumi364** [garmeeh/next-seo#1106]
Describe the bug
ArticleJson does not accept additional custom prop from schema, as other schema can accept additional extra props.

Reproduction
To produce you need to add custom prop in ArticleJson Ld

Expected behavior
Accepted behavior is we should see the additional prop in the schema

For reference
I think {...props} is missing here also like previously breadcrumb json's one.
![Screenshot_1](https://user-images.githubusercontent.com/16230189/203242584-62dd4eea-2ab2-420b-9b41-b9a74a08a642.png)

**Comment by Kyoss79** [comment]
Yes, please allow at least authorUrl, since google keeps whining about it:

![Uploading Screenshot 2023-01-22 at 21.41.58.png…]()

**Comment by AniruddhaGawali** [comment]
Is this issue still open, I can help in that

**Comment by nehagupta1504** [comment]
I have checked the code it has been done 17 months ago. Please close this PR
<img width="575" alt="image" src="https://github.com/garmeeh/next-seo/assets/66299920/4fa07d72-a899-444f-bdc3-c7be6d087373">
